tests/meson.build

```diff
@@ -151,6 +151,7 @@ foreach t, extra_args : tests
       depends: [softoken_conf, softhsm_conf],
       env: test_env,
       is_parallel: is_parallel,
+      timeout: 300,
```

Comment:
Why a 5 minutes timeout?

Reply:
Do you know what is causing it to take so much time on your system?
On mine I can complete tests in a few seconds once setup is completed ...
The longest test on my laptop takes 5 seconds, most are sub-second

Reply:
> Do you know what is causing it to take so much time on your system? On mine I can complete tests in a few seconds once setup is completed ... The longest test on my laptop takes 5 seconds, most are sub-second

I always perform clean compile and test. If compilation is cached, I guess it should not be necessary

Reply:
Is this a particularly slow system?

This is my timing:
```
$ time (git clean -f -x -d -f; make; make check)
[..]
Ok:                 33  
Expected Fail:      0   
Fail:               0   
Unexpected Pass:    0   
Skipped:            0   
Timeout:            0   

Full log written to /home/remote/ssorce/devel/git/pkcs11-provider/builddir/meson-logs/testlog.txt

real	0m12.833s
user	0m14.241s
sys	0m2.443s
```

About 15s total including compilation from a clean tree

Reply:
> Is this a particularly slow system?

No, it isn´t:
```bash
$ lscpu
Architecture:             x86_64
  CPU op-mode(s):         32-bit, 64-bit
  Address sizes:          39 bits physical, 48 bits virtual
  Byte Order:             Little Endian
CPU(s):                   8
  On-line CPU(s) list:    0-7
Vendor ID:                GenuineIntel
  Model name:             Intel(R) Core(TM) i7-10610U CPU @ 1.80GHz
...
```
> 
> This is my timing:
> 
> ```
> $ time (git clean -f -x -d -f; make; make check)
> [..]
> Ok:                 33  
> Expected Fail:      0   
> Fail:               0   
> Unexpected Pass:    0   
> Skipped:            0   
> Timeout:            0   
> 
> Full log written to /home/remote/ssorce/devel/git/pkcs11-provider/builddir/meson-logs/testlog.txt
> 
> real	0m12.833s
> user	0m14.241s
> sys	0m2.443s
> ```
> 
> About 15s total including compilation from a clean tree

I just removed everything and, although it is taking longer than yours, tests are passing correctly, 10 secs the longer one. My machine should be running some weird/heavy process ... I already removed the timeout change.